modules/ossm-upgrade-23-24-changes.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/upgrading-ossm.adoc
+
+:_content-type: CONCEPT
+[id="ossm-upgrade-23-24-changes_{context}"]
+= Upgrade changes from version 2.3 to version 2.4
+
+Upgrading the {SMProductShortName} control plane from version 2.3 to 2.4 introduces the following behavioral changes:
+
+* Support for Istio OpenShift Routing (IOR) has been deprecated. IOR functionality is still presently enabled, but it will be removed in a future release.
+
+* Support for the following cipher suites, which were deprecated in {SMProductName} 2.3, has been removed from the default list of ciphers used in TLS negotiations on both the client and server sides. Applications that require access to services requiring one of these cipher suites will fail to connect when a TLS connection is initiated from the proxy.
```

Comment:
```suggestion
* Support for the following cipher suites, which were deprecated in {SMProductName} 2.3, has been removed from the default list of ciphers used in TLS negotiations on both the client and server sides. Applications that require access to services requiring one of these cipher suites will fail to connect when a TLS connection is initiated from the proxy.
```
The wording at the start of this sentence is confusing: `Support... has been removed from the default list` Maybe just "the following cipher suites are no longer supported and are no longer on the list of..."

Reply:
Done